elastic/kibana
# [Remote clusters] add indexManagement as required plugin

This PR is continuation of https://github.com/elastic/kibana/pull/57365.

Now that `index_management` has been moved out of legacy, we can add it as a required plugin for `remote_clusters`. This is to provide the same functionality previously defined in legacy [here](https://github.com/elastic/kibana/blob/v7.6.0/x-pack/legacy/plugins/remote_clusters/index.ts#L41).

### How to test
1. Verify `remote_clusters` loads as expected with default config.
2. Update your `kibana.dev.yml` file to include: `xpack.index_management.enabled: false`. Verify the `remote_clusters` plugin does not load.
